docs/api/address/get-address-stx-balance.schema.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "GET request that returns address balances",
+  "title": "AddressStxBalanceResponse",
+  "type": "object",
+  "required": ["stx", "fungible_tokens", "non_fungible_tokens"],
```

Comment:
This doesn't quite work. The latter 2 aren't required for the stx balance, right? and also the emitted type comes out as non-required

Reply:
Better to require it the other way around. Make this it's own entity `stx-balance.schema.json`. and use an `allOf` in balance to merge a `$ref` to this, and the additional ft, nft schemas.